### DISCUSSION TITLE
Screen shots and Visual Architecture Diagram

### DISCUSSION THREAD

**Discussion by papyr** [mixcore/mix.core#433] [General Discussion]
Hello, this is a very nice approach to CMS. 

1. Visuals: The introduction or website would benefit with a demo page, or **some screen shots** on how easy it is to use.  I saw the youtube videos which is also nice 👍 

4. Also consider letting developers scaffold plugins or CRUD entities easily,

6. Marketplace for developers to list their plugins, themes

8. Manage tenants

10. Visual diagram with the components or overall architecture colored, for e.g. with green blue boxes on which ones the developer is coding and which components he is connecting/leveraging

**Comment by Smilefounder** [comment]
Thanks @papyr. Our team is trying super hard on v2 at the moment, we will improve our documentation, videos, guideline ASAP. Have a great day!

@nhathoang989
